include/alpaka/core/BoostPredef.hpp

```diff
@@ -57,17 +62,6 @@
 #    define BOOST_COMP_CLANG_CUDA BOOST_VERSION_NUMBER_NOT_AVAILABLE
 #endif
 
-// Intel compiler detection
-// As of Boost 1.74, Boost.Predef's compiler detection is a bit weird. Recent Intel compilers will be identified as
-// BOOST_COMP_INTEL_EMULATED. Boost.Predef has lackluster front-end support and mistakes the EDG front-end
-// for an actual compiler.
-// TODO: Whenever you look at this code please check whether https://github.com/boostorg/predef/issues/28 and
-// https://github.com/boostorg/predef/issues/51 have been resolved.
```

Comment:
The explanation here needs to be put in front of the PGI compiler detection now.

Reply:
done.